### PR TITLE
lang: Add to_int and to_float functions

### PR DESCRIPTION
**Pull request by dbakong**
## Tips:

* please read the style guide before submitting your patch:
[docs/style-guide.md](../docs/style-guide.md)

* commit message titles must be in the form:

```topic: Capitalized message with no trailing period```

or:

```topic, topic2: Capitalized message with no trailing period```

* golang code must be formatted according to the standard, please run:

```
make gofmt		# formats the entire project correctly
```

or format a single golang file correctly:

```
gofmt -w yourcode.go
```

* please rebase your patch against current git master:

```
git checkout master
git pull origin master
git checkout your-feature
git rebase master
git push your-remote your-feature
hub pull-request	# or submit with the github web ui
```

* after a patch review, please ping @purpleidea so we know to re-review:

```
# make changes based on reviews...
git add -p		# add new changes
git commit --amend	# combine with existing commit
git push your-remote your-feature -f
# now ping @purpleidea in the github PR since it doesn't notify us automatically
```

### CODE REVIEW

File: lang/funcs/convert/to_float_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	}
+	if got.Float() != expected {
+		t.Errorf("invalid output, expected %v, got %v", expected, got.Float())
+		return
+	}
+}
+
+func TestToFloat1(t *testing.T) {
+	testToFloat(t, 2, 2)
+}
```
Comment (dbakong) [inline lang/funcs/convert/to_float_test.go]: Not sure I need a test for the `ToFloat` function - almost feels like it is testing the functionality of `Float()`.
@purpleidea - Please let me know if you would recommend a more fitting test.

Comment (purpleidea) [reply lang/funcs/convert/to_float_test.go]: I think it's okay. I would probably write 2.0 for the float part to be more explicit though.

File: lang/funcs/convert/to_float.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+)
+
+func init() {
+	simple.ModuleRegister(ModuleName, "to_float", &types.FuncValue{
+		T: types.NewType("func(a int) float"),
+		V: ToFloat,
+	})
+}
+
+// ToFloat converts an integer to a float
```
Comment (purpleidea) [inline lang/funcs/convert/to_float.go]: Missing a period at the end. Same with other one :/

**Comment by purpleidea** [comment]
\o/ Merged!

**Pull request merged**